module3/projects/rancid_tomatillos/technical_requirements.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Rancid Tomatillos Technical Requirements
+module: 3
+tags: react,  testing, javascript, api, cypress
+---
+
+_[Back to Rancid Tomatillos Home](./index)_
+
+## Styling
+
+You do not have to match our styling for this project. That said, we want you to make intentional styling choices and for your final product to look polished. If you'd rather not spend time creating a design for this app, you may choose to match our design exactly. Pay close attention to the details - intentional hover states, spacing, and borders can make a huge difference. Be prepared to defend your styling choices in your evaluation with your instructor.
+
+## Iterations
+
+<section class="dropdown">
+### Iteration 0: Set Up & Planning
+
+You will be starting with this boilerplate repo: **[https://github.com/turingschool-examples/rancid-tomatillos-starter](https://github.com/turingschool-examples/rancid-tomatillos-starter)**. Follow the steps in the README carefully.
+
+
+**By the end of the day, Day 1:**
+* The link to your forked repo (should be renamed to NOT include "starter" in the name)
+* The link to your DTR
+
+**By the end of the day, Day 2:**
+* The link to your **[Project Board](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project)**. Your Project Board should include:
+  - At least 4 columns, i.e. `To Do`, `In Progress`, `In Review`, `Merged/Done`
+  - At least 6 tickets, with each ticket including:
+    - The associated user story from this project spec
+    - A **[label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)** that indicates the associated iteration (i.e. `iteration 1`)
+    - Some details in the description (a ticket's content should not just be the title)
+    - At least 2 tickets should be **[assigned](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)** to a person
+* An initial plan for your component architecture. Any format is fine, though you might find something like **[excalidraw](https://excalidraw.com/)** to be helpful. You should answer the following questions:
+  - What components do you envision needing? What is the relationship between these components (parents, siblings, etc)?
+  - Where will data be stored? Which components will need **state**?
+  - How will information be passed around? What **props** will be necessary? 
+
+<section class="note">
+### Turn in all deliverables via [this google sheet](https://docs.google.com/spreadsheets/d/1YDI4TAFdkOhpcFatdGONR7QtlDZVOpsvJVimtX77M6I/edit?usp=sharing).
```

Comment:
Ohhh I see. Can we make this note at the top of the section so that students don't send things to us directly?